tt_metal/hw/inc/wormhole/noc_nonblocking_api.h

```diff
@@ -292,6 +292,7 @@ inline __attribute__((always_inline)) void noc_fast_write_dw_inline(uint32_t noc
 
   uint32_t be32 = be;
   uint32_t be_shift = (dest_addr & (NOC_WORD_BYTES-1));
+  // If we're given a misaligned address, don't write to the bytes in the word below the address
```

Comment:
I only added the comment because I was confused at first about what was happening here. I can add the corresponding comment there too if it is common in those implementations

Reply:
I'll be honest, there isn't any documentation on these byte enable registers - I was told we need to look at the RTL to figure out exactly how it behaves, but from the line of code here, it is relative to the noc word size.

Reply:
Would you prefer if I reword it to say "An address not a multiple of noc word size". Again I don't know why this matters but really the comment is just to describe what the shift itself is trying to do. Maybe it's obvious in hindsight but I had to sit there for a minute and think about it before I realized what was going on.

Reply:
I see. I think either is fine, just wanted clarification since we typically only dealt with noc alignment